python-sdk/examples/openai_bot_prompt.py

```diff
@@ -0,0 +1,38 @@
+from dotenv import load_dotenv
+import os
+
+load_dotenv()
+
+from openai import OpenAI
+
+client = OpenAI()
+
+import langwatch.prompts
+import asyncio
+
+langwatch.setup(
+    api_key=os.getenv("LANGWATCH_API_KEY"),
+    endpoint_url=os.getenv("LANGWATCH_ENDPOINT"),
+)
```

Comment:
those are defaults, so you can shorten to this if you like

```suggestion
langwatch.setup()
```